lib/Launcher.js

```diff
@@ -270,17 +270,17 @@ class Launcher {
   }
 
   /**
-   * @param {!(BrowserOptions & {browserWSEndpoint: string})} options
+   * @param {!(BrowserOptions & {browserWSEndpoint: string, transport: ?Puppeteer.ConnectionTransport})} options
```

Comment:
nit: `transport?: !Puppeteer.ConnectionTransport` is a bit more precise. It isn't nullable, its only optional.

Reply:
Done.